tools/protoc-plugin/build.gradle.kts

```diff
@@ -54,4 +54,7 @@ tasks.jar {
             else -> zipTree(it)
         }
     })
+    // We should provide a classifier or else Protobuf Gradle plugin will substitute an OS-specific
```

Comment:
Sorry to bother you again on the same matter, but I think it's `will substitute it with an OS-specific one.`